StabQuest/Character.cs

```diff
@@ -76,16 +78,34 @@ public void Print()
 
         public int Level { get; set; }
 
-        public int CurrentExperience { get; set; }
-        public int ExperienceForNextLevel { get; set; }
+        public int CurrentExperience {
+            get { return _currentExperience; }
+            set {
+                _currentExperience  = value;
+
+                if(_currentExperience >= ExperienceForNextLevel)
```

Comment:
Might not be possible (yet), but what happens if you get so much exp that you should lvl up twice?

Reply:
Any ideas of how to fix? Maybe while?

Reply:
Fixed